scalding-commons/src/main/scala/com/twitter/scalding/commons/source/PailSource.scala

```diff
@@ -153,7 +169,6 @@ class PailSource[T] private (rootPath: String, structure: PailStructure[T], subP
   val fieldName = "pailItem"
 
   lazy val getTap = {
-    val spec = PailTap.makeSpec(null, structure)
     val javaSubPath = if ((subPaths == null) || (subPaths.size == 0)) null else subPaths map { _.asJava }
```

Comment:
Since the constructor setting up the null for the list is private, and we only seem to care if the path is empty or not. Can we just default subPaths to an empty list and not need to do null checking?